### DISCUSSION TITLE
Typsetting of inverse function

### DISCUSSION THREAD

**Discussion by deepimpactmir** [typst/typst#4041] [Q&A] [answered]
I want to write something like `$^{-1}$` but the "equivalence" `$^(-1)$` doesn't work in Typst. Is there any way I can do this in Typst?

**Comment by EpicEricEE** [comment, marked as answer]
There has to be something to attach the -1 to, so you can do `$#[]^(-1)$`.

**Reply by deepimpactmir** [reply]
That worked! Thanks!
